conf/default.toml

```diff
@@ -76,6 +76,10 @@ primary_color = "#c82850"
 secondary_color = "#001638"
 main_color = "#212029" # All-purpose text color
 background_color = "#fdfbff"
+# Title colors can be defined optionally. Make use of title_header_color if a higher contrast to the header background is needed.
+# If not set, both values default to black.
+# title_color = '#000000'
+# title_header_color = '#000000'
```

Comment:
IMO these settings are confusing.
* Title color feels like it does not fit in our concept of theme, as I wrote in an earlier comment
* Title header color is definitely relevant, but it should be placed closer to the `header_background` setting since the reason it exists is to allow users to define an "opposite" or "accent" color alongside a header background; I would call it `header_foreground_color` and put it right underneath the `header_background` setting, specify it is optional (default to white) and that it should be defined according to the header background

Reply:
Good idea, I'll adapt this, thanks!